lib/web_console/templates/console.js.erb

```diff
@@ -490,13 +567,15 @@ REPLConsole.prototype.renderInput = function() {
   }
 
   this.promptDisplay.appendChild(document.createTextNode(before));
-  promptCursor.appendChild(document.createTextNode(current));
-  this.promptDisplay.appendChild(promptCursor);
+  this.promptDisplay.appendChild(center);
   this.promptDisplay.appendChild(document.createTextNode(after));
+
+  var hint = this.autocomplete && !this.autocomplete.confirmed && this.autocomplete.getSelectedWord();
```

Comment:
I see we check for autocomplete presence quite often. Can we somehow have an internal flag that turns methods into noops if we are `disabled` or `not confirmed`?